framework/src/main/java/com/jfcbuilder/builders/IXYTimeSeriesBuilder.java

```diff
@@ -38,7 +38,7 @@
  * @param <T> The method chaining return type, which must be the type of the builder implementing
  *        this interface.
  */
-interface IXYTimeSeriesBuilder<T extends IXYTimeSeriesBuilder<T>> {
+public interface IXYTimeSeriesBuilder<T extends IXYTimeSeriesBuilder<T>> {
```

Comment:
IXYTimeSeriesPlotBuilder still has old references to "line builder". Should be "marker builder"